tests/post_processing/test_single_file_post_processing.py

```diff
@@ -132,6 +132,40 @@ def test_calculate_probabilities_and_thresholds_success():
         prob_and_threshold_data, [test_input["measure_loc"]]
     )
 
+def test_calculate_probabilities_and_thresholds_success():
+    test_input = generate_test_input()
+    prob_and_threshold_data = (
+        single_file_post_processing._calculate_probabilities_and_thresholds(
+            filtered_model_outputs=test_input["input"],
+            year_column_loc=test_input["year_loc"],
+            measure_column_loc=test_input["measure_loc"],
+            age_start_column_loc=test_input["age_start_loc"],
+            age_end_column_loc=test_input["age_end_loc"],
+            draws_loc=test_input["draws_loc"],
+            prevalence_marker_name=PREV_MEASURE_NAME,
+            threshold=0.1,
+            pct_runs_under_threshold=[0.8, 0.9],
+        )
+    )
+    assert (
```

Comment:
❓ I'm struggling to follow this test to be sure it is correct. In particular I can't see how we know it has indeed worked out the right values for 0.8 and 0.9.